ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x03D62C00>, trait: plot_options, old value: None, new value: 1.0 0.0 1.0 0.0 [101, 101] False False
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 64, in update_binning
    self.data_container.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x03D62C00>, trait: plot_options, old value: None, new value: 1.0 0.0 1.0 0.0 [101, 101] False False
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 58, in update_range
    self.plot.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x03D62C00>, trait: thresholds, old value: None, new value: inf 0.0 inf 0.0 1.0 1.0 0.0 0.0 1.0 0.8 0.0 0.3 inf 0.0 inf 250.0
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 74, in apply_thresholds
    self.data_container._apply_thresholds()
AttributeError: 'NoneType' object has no attribute '_apply_thresholds'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x03D62C00>, trait: evss_layers, old value: None, new value: True False True True True
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 114, in redraw_plot
    self.plot.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x03D62C00>, trait: corr, old value: None, new value: 0.0 False 0.0 0.0 1.0 0.0 0.0 0.0 1.0
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 152, in apply_corrections
    self.data_container.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: 0.0 False 0.0 0.0 1.0 0.0 0.0 0.0 1.0, trait: thorben_hack, old value: None, new value: <undefined>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 127, in script_bg
    photons = self.data_manager.datasets[0].get_photons()
AttributeError: 'NoneType' object has no attribute 'datasets'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x03E203C0>, trait: data_container, old value: None, new value: <alex.models.EvsS object at 0x03EF0D20>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\views.py", line 602, in set_show_S_range
    self.evss_layers.show_S_range = not (self.data_container.thresholds.S_max_1D == self.data_container.plot_options.S_max and self.data_container.thresholds.S_min_1D == self.data_container.plot_options.S_min)
AttributeError: 'NoneType' object has no attribute 'show_S_range'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x03E203C0>, trait: data_container, old value: None, new value: <alex.models.EvsS object at 0x03EF0D20>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\views.py", line 609, in set_show_E_range
    self.evss_layers.show_E_range = not (self.data_container.thresholds.E_max_1D == self.data_container.plot_options.E_max and self.data_container.thresholds.E_min_1D == self.data_container.plot_options.E_min)
AttributeError: 'NoneType' object has no attribute 'show_E_range'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x03E203C0>, trait: rectangle, old value: None, new value: <undefined>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\controller.py", line 427, in add_guess_rect
    x1 = rect[0][0]
TypeError: '_Undefined' object does not support indexing
WARNING:enaml.session:Invalid object id sent to Session: o_2w:index_changed
WARNING:enaml.session:Invalid object id sent to Session: o_2w:index_changed
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x01FA8A50>, trait: plot_options, old value: None, new value: 1.0 0.0 1.0 0.0 [101, 101] False False
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 64, in update_binning
    self.data_container.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x01FA8A50>, trait: plot_options, old value: None, new value: 1.0 0.0 1.0 0.0 [101, 101] False False
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 58, in update_range
    self.plot.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x01FA8A50>, trait: thresholds, old value: None, new value: inf 0.0 inf 0.0 1.0 1.0 0.0 0.0 1.0 0.5 0.0 0.22 inf 0.0 inf 150.0
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 74, in apply_thresholds
    self.data_container._apply_thresholds()
AttributeError: 'NoneType' object has no attribute '_apply_thresholds'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x01FA8A50>, trait: evss_layers, old value: None, new value: True False True True True
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 114, in redraw_plot
    self.plot.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x01FA8A50>, trait: corr, old value: None, new value: 0.0 False 0.0 0.0 1.0 0.0 0.0 0.0 1.0
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 152, in apply_corrections
    self.data_container.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: 0.0 False 0.0 0.0 1.0 0.0 0.0 0.0 1.0, trait: thorben_hack, old value: None, new value: <undefined>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 127, in script_bg
    photons = self.data_manager.datasets[0].get_photons()
AttributeError: 'NoneType' object has no attribute 'datasets'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x0AADBB40>, trait: data_container, old value: None, new value: <alex.models.EvsS object at 0x0AADB390>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\views.py", line 602, in set_show_S_range
    self.evss_layers.show_S_range = not (self.data_container.thresholds.S_max_1D == self.data_container.plot_options.S_max and self.data_container.thresholds.S_min_1D == self.data_container.plot_options.S_min)
AttributeError: 'NoneType' object has no attribute 'show_S_range'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x0AADBB40>, trait: data_container, old value: None, new value: <alex.models.EvsS object at 0x0AADB390>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\views.py", line 609, in set_show_E_range
    self.evss_layers.show_E_range = not (self.data_container.thresholds.E_max_1D == self.data_container.plot_options.E_max and self.data_container.thresholds.E_min_1D == self.data_container.plot_options.E_min)
AttributeError: 'NoneType' object has no attribute 'show_E_range'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x0AADBB40>, trait: rectangle, old value: None, new value: <undefined>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\controller.py", line 427, in add_guess_rect
    x1 = rect[0][0]
TypeError: '_Undefined' object does not support indexing
WARNING:enaml.session:Invalid object id sent to Session: o_jL:index_changed
WARNING:enaml.session:Invalid object id sent to Session: o_jL:index_changed
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0A5D0DE0>, trait: plot_options, old value: None, new value: 1.0 0.0 1.0 0.0 [101, 101] False False
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 64, in update_binning
    self.data_container.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0A5D0DE0>, trait: plot_options, old value: None, new value: 1.0 0.0 1.0 0.0 [101, 101] False False
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 58, in update_range
    self.plot.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0A5D0DE0>, trait: thresholds, old value: None, new value: inf 0.0 inf 0.0 1.0 1.0 0.0 0.0 1.0 0.5 0.0 0.22 inf 0.0 inf 150.0
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 74, in apply_thresholds
    self.data_container._apply_thresholds()
AttributeError: 'NoneType' object has no attribute '_apply_thresholds'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0A5D0DE0>, trait: evss_layers, old value: None, new value: True False True True True
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 114, in redraw_plot
    self.plot.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0A5D0DE0>, trait: corr, old value: None, new value: 0.0 False 0.0 0.0 1.0 0.0 0.0 0.0 1.0
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 152, in apply_corrections
    self.data_container.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: 0.0 False 0.0 0.0 1.0 0.0 0.0 0.0 1.0, trait: thorben_hack, old value: None, new value: <undefined>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 127, in script_bg
    photons = self.data_manager.datasets[0].get_photons()
AttributeError: 'NoneType' object has no attribute 'datasets'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x0BD08030>, trait: data_container, old value: None, new value: <alex.models.EvsS object at 0x0BD0F540>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\views.py", line 602, in set_show_S_range
    self.evss_layers.show_S_range = not (self.data_container.thresholds.S_max_1D == self.data_container.plot_options.S_max and self.data_container.thresholds.S_min_1D == self.data_container.plot_options.S_min)
AttributeError: 'NoneType' object has no attribute 'show_S_range'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x0BD08030>, trait: data_container, old value: None, new value: <alex.models.EvsS object at 0x0BD0F540>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\views.py", line 609, in set_show_E_range
    self.evss_layers.show_E_range = not (self.data_container.thresholds.E_max_1D == self.data_container.plot_options.E_max and self.data_container.thresholds.E_min_1D == self.data_container.plot_options.E_min)
AttributeError: 'NoneType' object has no attribute 'show_E_range'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x0BD08030>, trait: rectangle, old value: None, new value: <undefined>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\controller.py", line 427, in add_guess_rect
    x1 = rect[0][0]
TypeError: '_Undefined' object does not support indexing
WARNING:enaml.session:Invalid object id sent to Session: o_rq:index_changed
WARNING:enaml.session:Invalid object id sent to Session: o_rq:index_changed
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0C1220C0>, trait: plot_options, old value: None, new value: 1.0 0.0 1.0 0.0 [101, 101] False False
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 64, in update_binning
    self.data_container.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0C1220C0>, trait: plot_options, old value: None, new value: 1.0 0.0 1.0 0.0 [101, 101] False False
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 58, in update_range
    self.plot.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0C1220C0>, trait: thresholds, old value: None, new value: inf 0.0 inf 0.0 1.0 1.0 0.0 0.0 1.0 0.5 0.0 0.22 inf 0.0 inf 150.0
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 74, in apply_thresholds
    self.data_container._apply_thresholds()
AttributeError: 'NoneType' object has no attribute '_apply_thresholds'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0C1220C0>, trait: evss_layers, old value: None, new value: True False True True True
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 114, in redraw_plot
    self.plot.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.controller.EvsSViewController object at 0x0C1220C0>, trait: corr, old value: None, new value: 0.0 False 0.0 0.0 1.0 0.0 0.0 0.0 1.0
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 152, in apply_corrections
    self.data_container.update()
AttributeError: 'NoneType' object has no attribute 'update'
ERROR:traits:Exception occurred in traits notification handler for object: 0.0 False 0.0 0.0 1.0 0.0 0.0 0.0 1.0, trait: thorben_hack, old value: None, new value: <undefined>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\mixins.py", line 127, in script_bg
    photons = self.data_manager.datasets[0].get_photons()
AttributeError: 'NoneType' object has no attribute 'datasets'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x0C119750>, trait: data_container, old value: None, new value: <alex.models.EvsS object at 0x0C119C30>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\views.py", line 602, in set_show_S_range
    self.evss_layers.show_S_range = not (self.data_container.thresholds.S_max_1D == self.data_container.plot_options.S_max and self.data_container.thresholds.S_min_1D == self.data_container.plot_options.S_min)
AttributeError: 'NoneType' object has no attribute 'show_S_range'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x0C119750>, trait: data_container, old value: None, new value: <alex.models.EvsS object at 0x0C119C30>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\views.py", line 609, in set_show_E_range
    self.evss_layers.show_E_range = not (self.data_container.thresholds.E_max_1D == self.data_container.plot_options.E_max and self.data_container.thresholds.E_min_1D == self.data_container.plot_options.E_min)
AttributeError: 'NoneType' object has no attribute 'show_E_range'
ERROR:traits:Exception occurred in traits notification handler for object: <alex.views.EvsSPlot object at 0x0C119750>, trait: rectangle, old value: None, new value: <undefined>
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 520, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "C:\Python27\lib\site-packages\traits\trait_notifiers.py", line 483, in dispatch
    handler( *args )
  File "C:\Documents and Settings\confocal\Desktop\ALEX-Suite-1.2.0\alex\controller.py", line 427, in add_guess_rect
    x1 = rect[0][0]
TypeError: '_Undefined' object does not support indexing
WARNING:enaml.session:Invalid object id sent to Session: o_xF:index_changed
WARNING:enaml.session:Invalid object id sent to Session: o_xF:index_changed
